test/Interpreter/Inputs/ObjCClasses/ObjCClasses.h

```diff
@@ -94,6 +94,11 @@ NS_ASSUME_NONNULL_BEGIN
 - (id) initWithArray: (NSArray<T>*) array;
 @end
 
+@interface TestingBool : NSObject
+- (void) shouldBeTrueObjCBool: (BOOL)value;
```

Comment:
Nitpick: Apple Objective-C style uses spaces as follows:

```objc
- (void)doTheThing:(id)foo toOtherThing:(id)bar;
```

I see that the rest of the file is wrong too, though.